composer.json

```diff
@@ -34,15 +34,13 @@
     "phpstan/phpstan": "^1.10.47",
     "phpstan/phpstan-deprecation-rules": "^1.1",
     "phpstan/phpstan-doctrine": "^1.3.40",
-    "phpstan/phpstan-php-parser": "^1.1",
```

Comment:
This is only useful once we're writing own code in php-parser. Now we can drop it as unused